src/loaders/FileLoader.js

```diff
@@ -184,13 +184,12 @@ Object.assign( FileLoader.prototype, {
 			if ( this.responseType !== undefined ) request.responseType = this.responseType;
 			if ( this.withCredentials !== undefined ) request.withCredentials = this.withCredentials;
 
-			if ( ! this.mimeType && this.responseType !== 'blob' ) {
+			// Setting 'text/plain' as the MIME type for SVG files will cause a
+			// problem when the ImageLoader tries to use it as the src of an image,
+			// so explicitly set the SVG MIME type.
+			if ( /\.svg$/.test( url ) ) this.mimeType = 'image/svg+xml';
```

Comment:
OK, will update.

Just wondering why you prefer not to remove the `text/plain` override?